openshift_resource_quota.py

```diff
@@ -0,0 +1,101 @@
+import kubernetes
+import pprint
+import logging
+import requests
+import json
+import re
+from flask import Flask, redirect, url_for, request, Response
+
+import sys
+
+application = Flask(__name__)
+
+
+
+def exists_openshift_resource_quota(token,api_url,project_name,resource_name):
+    headers = {'Authorization': 'Bearer ' + token, 'Accept': 'application/json', 'Content-Type': 'application/json'}
+    url = 'https://' + api_url + '/api/v1/namespaces/' + project_name + '/resourcequotas/'+resource_name
+    r = requests.get(url, headers=headers, verify=False)
+    application.logger.debug("url: "+url)
+    application.logger.debug("cr p: " + str(r.status_code))
+    application.logger.debug("cr p: " + r.text)
+    if(r.status_code == 200 or r.status_code == 201):
+        return True
+    return False
```

Comment:
just return whatever is inside the if block, no need of `if` statements

Reply:
Ok. Will do that.

Reply:
you could further make it more pythonic by doing `r.status_code in [200, 201]` instead of having multiple separate comparisons.

Reply:
Sure, Thank You so much.